### ISSUE TITLE
Doc comments in the generated classes from .drift files

### ISSUE THREAD

**Issue by enrique-lozano** [simolus3/moor#2416]
I don't now if this is already possible somehow (I can not find any docs about that), but would be nice to have the possibility to document our attributes directly from the .drift files. Something like this:
![image](https://github.com/simolus3/drift/assets/61509169/bd51d145-48ba-4246-8aba-5a846cd35b8d)
Would generate the doc comment:
![image](https://github.com/simolus3/drift/assets/61509169/30309cce-ee54-4a35-92af-131c206e37c0)

**Comment by simolus3** [comment]
I like the idea and I've implemented it in 18868221d936e483a57bc0ec1214fcd5245d9250.

But I think the comment should go before the column (and we'll only pick that up). Just like you can't do this in Dart:

```dart
  final String code; /// A doc comment
```

I think a more intuitive syntax for documentation comments in SQL would be

```sql
CREATE TABLE currencies (
  -- A doc comment example
  code TEXT NOT NULL PRIMARY KEY,
  symbol TEXT NOT NULL
) AS Currency;
```

**Comment by enrique-lozano** [comment]
Yep, it's much more intuitive that way.

Thanks a lot! Faster impossible, tremendous work!
